ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -91,6 +91,7 @@ Query.Model = Backbone.AssociatedModel.extend({
         type: 'text',
         isLocal: false,
         isOutdated: false,
+        hasBeenSaved: false,
```

Comment:
✏️ Maybe this should be called `saved` to match `Workspace`'s `saved` property?

Reply:
I think `hasBeenSaved` is more descriptive as to what this is intending to flag. As in if you just created a query `hasBeenSaved`? => false